ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-rollback-nightly.yaml

```diff
@@ -42,6 +42,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: 4.17.0-rc.1
+  source:
```

Comment:
The rollback nightly ci does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-hypershift-rollback which has pre step, so keep it.